nessus_parameterstorereadonly_policy.tf

```diff
@@ -0,0 +1,25 @@
+# ------------------------------------------------------------------------------
+# Create the IAM policy that allows read-only access to the Nessus-related
+# SSM Parameter Store parameters in the Images account.
+# ------------------------------------------------------------------------------
+
+data "aws_iam_policy_document" "nessus_parameterstorereadonly_doc" {
+  statement {
+    actions = [
+      "ssm:GetParameter",
+      "ssm:GetParameters"
+    ]
+    resources = [
+      format("arn:aws:ssm:*:%s:parameter%s", local.images_account_id, var.ssm_key_nessus_admin_username),
```

Comment:
`arn:aws:ssm:${local.images_account_id}:parameter${var.ssm_key_nessus_admin_username}` will also work here.  I'm fine with either syntax; I just wanted to point out that `format()` isn't required here.

Reply:
I like the non-`format()` syntax a bit more, so I made the change in https://github.com/cisagov/cool-assessment-terraform/pull/31/commits/2abd89edf316433eff25bfb4d5ecbed3c43dc8c4.  Anything that is more "f-string-like" is good by me!  Thanks for mentioning this. 👍